README.md

```diff
@@ -131,7 +133,7 @@ Use the images locally
 You can also use the images locally to build or test packages, this is an example command:
 
 ```
-docker run -v/tmp/.conan:/home/conan/.conan conanio/gcc63 bash -c "conan install zlib/1.2.11@conan/stable --build missing"
+docker run -v/tmp/.conan:/home/conan/.conan conanio/gcc63 bash -c "conan install lz4/1.8.3@bincrafters/stable --build missing"
```

Comment:
Why this change?

Reply:
Good question. I started this branch days ago when we had a problem downloading zlib from bintray. The goal is the same, build a C project. LZ4 is smaller than zlib thus our tests will be faster than before